mods/cnc/maps/nod01/map.yaml

```diff
@@ -296,8 +296,10 @@ Rules:
 		Invulnerable:
 	^CivBuilding:
 		MustBeDestroyed:
+			RequiredForShortGame: true
```

Comment:
These don't seem to be related to the upgrade rule either.  It would be best if these could live in their own commit.